website/frontend/src/assets/scss/index.scss

```diff
@@ -13,6 +13,7 @@ html {
 body {
   background: #282828;
   font-size: 14px;
+  max-width: calc(100vw - 20px);
```

Comment:
we could try to use this approach https://ishadeed.com/article/custom-scrollbars-css/#scrollbar-gutter

Reply:
It unclear, in article is about custom scrollbars, but this css fix shaknig